tasklist/src/pages/listTask/listTask.html

```diff
@@ -18,6 +18,38 @@
 </ion-header>
 
 <ion-content>
+  <ion-item>
+    <ion-searchbar [(ngModel)]="searchName" (ionInput)="getDataFromApi()"></ion-searchbar>
+  </ion-item>
+  <ion-item>
+    <ion-label>Id</ion-label>
+    <ion-input [(ngModel)]="searchId" (input)="getDataFromApi()"></ion-input>
+  </ion-item>
+  <ion-item>
+    <ion-label>Status</ion-label>
+    <ion-select [(ngModel)]="searchDone" (ionChange)="getDataFromApi()">
+      <ion-option value="all" selected="true">All</ion-option>
+      <ion-option value="done">Done</ion-option>
```

Comment:
It will be good to keep the same label for User Experience. The user only know `open` and `closed`